src/Tribe/Cache.php

```diff
@@ -536,4 +536,93 @@ public function data_size_over_packet_size( $value ) {
 		// If the size of the string is above 90% of the database `max_allowed_packet` setting, then it should not be written to the db.
 		return $size > ( $feature_detection->get_mysql_max_packet_size() * .9 );
 	}
+
+	/**
+	 * Returns a transient that might have been stored, due ot its size, in chunks.
+	 *
+	 * @since TBD
+	 *
+	 * @param string               $id                 The name of the transients to return.
+	 * @param string|array<string> $expiration_trigger The transient expiration trigger(s).
+	 *
+	 * @return false|mixed Either the transient value, joined back into one, or `false` to indicate
+	 *                     the transient was not found or was malformed.
+	 */
+	public function get_chunkable_transient( $id, $expiration_trigger = '' ) {
+		$transient = $this->get_id( $id, $expiration_trigger );
+
+		if ( wp_using_ext_object_cache() ) {
+			return get_transient( $transient );
+		}
+
+		$chunks = [];
+		$i      = 0;
```

Comment:
```suggestion
		$i      = 1;
```

As this is just an indicator for the transient number I think `1` makes more sense as start point